README.md

```diff
@@ -1,2 +1,79 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Collaborative Project: The Next Web
+
+> This is a microverse student project for working with hierarchy of elements and responsive design
+
+![screenshot](https://github.com/Gnwin/Godwin-Eraani-Next-Web-project/blob/nextweb/assets/images/Screenshotnewtnw.png)
+
+> This project clones the thenextweb.com article page interface design.
+> The project includes the header, the main content, footer, media queries.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Gnwin/Godwin-Eraani-Next-Web-project/nextweb/index.html)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> Browser
+> Text editor of your choice.
+> npm has to be installed in your system.
+> install linters on your IDE.
```

Comment:
If installing linters is a requirement to be able to check your project locally (on one's computer), please elaborate those linters that are needed. I mean list them hiere.